suite-native/module-device-settings/src/screens/ContinueOnTrezorScreen.tsx

```diff
@@ -21,18 +22,14 @@ export const ContinueOnTrezorScreen = () => {
 
     const deviceModel = useSelector(selectDeviceModel);
 
-    if (!deviceModel) {
-        return null;
-    }
-
     return (
         <DeviceInteractionScreenWrapper>
             <Text variant="titleMedium" style={applyStyle(titleStyle)}>
                 <Translation id="device.title.continueOnTrezor" />
             </Text>
             <Box flex={1} alignItems="center" justifyContent="flex-end">
                 <DeviceImage
-                    deviceModel={deviceModel}
+                    deviceModel={deviceModel || DeviceModelInternal.T3T1}
```

Comment:
When is it possible for `device` to be `null` and why do you fallback to this specific model?

Reply:
It should never happen but selectors uses `find` method which could return null so it needs to be handled. Picked random device as fallback since it in standart cirumstances it should never happen.